templates/thank-you.html

```diff
@@ -1,11 +1,38 @@
 {% extends 'layouts/_base.html' %}
+{% from 'components/panel/_macro.njk' import onsPanel %}
 
-{% set page_title = _("We\'ve received your answers") + " - " + page_header.title %}
+{% set page_title = _("We’ve received your answers") + " - " + page_header.title %}
 
 {% block main %}
+  {% if schema_theme == "default" %}
```

Comment:
I've just noticed there are two other themes - `social` and `northernireland` that will need to be considered. There are no current social surveys, but there are some `northernireland` ones.

Reply:
Fixed, thanks!